### DISCUSSION TITLE
Measure time

### DISCUSSION THREAD

**Discussion by CA10C3SAR** [zero-plusplus/vscode-autohotkey-debug#114] [Q&A] [answered]
While debugging there's any way to measure the time from a breakpoint to another?

**Comment by zero-plusplus** [comment, marked as answer]
[PerfTips](https://github.com/zero-plusplus/vscode-autohotkey-debug#perftips-optional) can be used to measure the time.

**Reply by CA10C3SAR** [reply]
Thank you got it, but it only works when i select "AutoHotkey Debugger"
![image](https://user-images.githubusercontent.com/6724006/127759448-0b046a16-ae73-427d-9cbd-14452d76d084.png)

By default when i press F5, vscode uses the "AutoHotkey Debugger Workspace" do you know how i could change it?

**Reply by zero-plusplus** [reply]
Click on `AutoHotkey Debugger` in the image to switch the debug settings.

---

Note that PerfTips is an optional setting, so you will need to edit launch.json.

If you know how to use the watch expression, you can substitute it by registering `{elapsedTime_s}`. This one will work with the default settings.
